lib/Hand.js

```diff
@@ -1,8 +1,15 @@
 
 import Finger from './Finger.js';
-import * as enums from './Enums.js';
 import * as capabilities from './Capabilities.js';
 
+const TimingMode = {
+	Interval: 'INTERVAL',
```

Comment:
Do we want to use `Symbol`s here? 
(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol)

Reply:
I was considering this, but aside from the fact that they are globally unique, they don't really add much. If you like I can add it regardless, I don't feel strongly either way.